test/functional/notes_controller_test.rb

```diff
@@ -690,6 +690,8 @@ def teardown
     get :recent
     notes = assigns(:notes)
     expected = [nodes(:one)]
+    # Just that using nodes(:question) is misleading because
+    # the /notes page is designed not to return any question related notes
     questions = [nodes(:question)]
```

Comment:
# Just that using nodes(:question) is misleading because
    # the /notes page is designed not to return any question related notes